Update README and rename directories in line with new terminology

gaurav opened the pull request:
Since the name and role of this repository has changed (#19), we no longer refer to these as testcases but as PHYX files. I've updated the README file to reflect this (closes #30) and renamed some of the directories in this repository to make their purpose clearer, as well as cleaning out some out-of-date information that no longer needs to be stored outside the PHYX file itself. Individual lists of phyloreferences have been removed, and will eventually be replaced by an automatically generated table that provides information on the types of included phyloreferences (#34).

hlapp commented on README.md:
```diff
@@ -1,25 +1,24 @@
-2. It provides a test space for trying different approaches to generating phyloreferences from JSON to OWL, although these will be moved into their own repositories if they prove to be successes.
-3. It provides a test suite of phyloreferences along with expected resolved nodes, allowing reasoning to be continually tested as ontologies and software tools are updated.
+The Clade Ontology is an ontology of exemplar phyloreferences curated from peer-reviewed publications. Phyloreferences in this ontology include their verbatim clade definition and the phylogeny upon which they were initially defined. The ontology therefore acts as both a catalogue of computable clade definitions as well as a test suite of phyloreferences that can be tested to determine if each ph ...
 
```
Where's the Apache license coming from?? OBO recommends CC-BY. I would suggest CC0, unless we have good arguments as to why we cannot waive copyright (for example, because someone other than us has it), or why our terms of reuse have to include a legal requirement for attribution (as opposed to a scientific norm mandating proper citation).

gaurav replied:
The Apache license is intended to apply to the software in this repository -- the phyx2owl script and the testing scripts -- which is clearly not what the text in the README file says. I've updated the README file to clarify that, and added a copy of the CC0 legal code into the `phyx` directory to make it clear that a separate license applies there.

As for the question of other authors' copyright, I don't think you could argue that the phylogeny (which is a series of assertions) or the raw phyloreference (which is a list of specifiers) count as creative works, so CC0 sounds great to me. We are currently recording information that helps to explain why the clade was defined in a particular way (e.g. "Because sister group relationships are better resolved and supported than basally-branching ingroup relationships within this clade, a branch-modified node-based definition was used to maximize definitional and compositional stability (Cantino et al., 2007).", see reference below). This is arguably copyrightable, but I think it's small enough that we could argue fair use or remove it on a case-by-case basis if there are specific complaints.

https://github.com/phyloref/clade-ontology/blob/187c78cdf71c163075f3a216e6f4c5caa46c3e8d/testcases_too_slow/Wojciechowski%2C%202013/paper.json#L74)

hlapp replied:
So fair use permitting us to reuse the definition texts here does not mean they are in the public domain, and fair use does mean you can claim intellectual property rights. So I'm skeptical that we have the rights to release the clade definition texts into the public domain.

Maybe this needs some consultation first with corresponding experts. Given the legal uncertainty, I'd argue we cannot at this point attach a CC0 waiver if the definition texts are included, because once we do we've waived our rights and we can't just take that back.

gaurav commented:
@hlapp I've tried to improve the licensing situation to distinguish between Apache licensed software (e.g. phyx2owl and the testing scripts) and the CC0 licensed ontology (the PHYX file and, eventually, the ontology), and fixed the obviously wrong text in the README file. Let me know if there are any additional changes I should make!

ncellinese commented:
Yes, totally agree with this.

nico


> On Jun 22, 2018, at 10:25 AM, Hilmar Lapp <notifications@github.com> wrote:
> 
> So fair use permitting us to reuse the definition texts here does not mean they are in the public domain, and fair use does mean you can claim intellectual property rights. So I'm skeptical that we have the rights to release the clade definition texts into the public domain.
> 
> Maybe this needs some consultation first with corresponding experts. Given the legal uncertainty, I'd argue we cannot at this point attach a CC0 waiver if the definition texts are included, because once we do we've waived our rights and we can't just take that back.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub <https://github.com/phyloref/clade-ontology/pull/35#issuecomment-399460617>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ACaXwRwl6nUA51VlKUNhZ_JaDhoB8EfNks5t_P5mgaJpZM4UsYCZ>.
>

hlapp commented on README.md:
```diff
@@ -1,25 +1,24 @@
-2. It provides a test space for trying different approaches to generating phyloreferences from JSON to OWL, although these will be moved into their own repositories if they prove to be successes.
-3. It provides a test suite of phyloreferences along with expected resolved nodes, allowing reasoning to be continually tested as ontologies and software tools are updated.
+The Clade Ontology is an ontology of exemplar phyloreferences curated from peer-reviewed publications. Phyloreferences in this ontology include their verbatim clade definition and the phylogeny upon which they were initially defined. The ontology therefore acts as both a catalogue of computable clade definitions as well as a test suite of phyloreferences that can be tested to determine if each ph ...
 
```
My feeling is we should standardize on a "default" software license, because I don't see much point in having different licenses for different pieces of software we develop, given that we're not defaulting on the GPL, which we may otherwise have reason to relax from depending on the specific tool or code.

So are you suggesting the Apache 2.0 license as our default, and do you have reasons to choose it over, say MIT, or BSD2/3, as the most commonly used "permissive" licenses.

gaurav replied:
Actually, it looks we've standardized to the MIT license across all Phyloref software products as we decided to in phyloref/phylo2owl#1 -- *except* for the Clade Ontology, which ended up with the Apache License before we'd closed that issue. I've now relicensed it with the MIT license in 5e1ec27.

gaurav commented:
I've moved all the links to the bottom of the README file and replaced the Apache license with an MIT license to bring it in line with all the other Phyloref software products. Let me know what you think!
